web-frame/src/assets/i18n/en.json

```diff
@@ -57,13 +71,20 @@
       "SUBMIT": "Login",
       "CREATE_ACCOUNT": "Create account",
       "FORGOT_PASSWORD": "Forgot password?",
-      "ERRORS": {
+      "ERROR": {
+        "EMAIL": {
+          "email": "Email is invalid.",
```

Comment:
Update to uppercase

Reply:
We can't really do this. These are translation tokens contain values from `ValidationErrors` instances and we don't control those. We could use a `snakeCase` function, but then it would make searching for these errors a bit hard. I would keep this as an exception, at least for now.